swapi/src/schema/schema.js

```diff
@@ -90,17 +117,17 @@ function rootConnection(name, swapiType) {
 var rootType = new GraphQLObjectType({
   name: 'Root',
   fields: () => ({
-    allFilms: rootConnection('Films', 'films'),
+    filmConnection: rootConnection('Films', 'films'),
```

Comment:
although allFilmsConnection is strictly more consistent I think I prefer allFilms